src/js/setUpSite.ts

```diff
@@ -119,12 +134,15 @@ const generateScorecard = (scoreCardEntry) => {
  * @param cityId: E.g. `columbus-oh`.
  * @param parkingLayer: GeoJSON layer with parking lot data
  */
-const loadParkingLot = async (cityId, parkingLayer) => {
+const loadParkingLot: (
+  cityId: CityId,
+  parkingLayer: GeoJSON<GeoJsonProperties, Geometry>
+) => Promise<void> = async (cityId, parkingLayer) => {
```

Comment:
Style nit. This will probably need reformatting though

```suggestion
const loadParkingLot = async (cityId: CityId, parkingLayer: GeoJSON<GeoJsonProperties, Geometry>): Promise<void> => {
```

Reply:
Ditto the other functions in this file like snapToCity